src/main/java/ch/njol/skript/expressions/ExprTimePlayed.java

```diff
@@ -18,68 +18,66 @@
  */
 package ch.njol.skript.expressions;
 
-import ch.njol.skript.classes.Changer.ChangeMode;
-import org.bukkit.OfflinePlayer;
-import org.bukkit.Statistic;
-import org.bukkit.entity.Player;
-import org.bukkit.event.Event;
-import org.eclipse.jdt.annotation.Nullable;
-
 import ch.njol.skript.Skript;
-import ch.njol.skript.classes.Changer;
+import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.Timespan;
-import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.Statistic;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Time Played")
 @Description("The amount of time a player has played for on the server. This info is stored in the player's statistics in " +
 	"the main world's data folder. Changing this will also change the player's stats which can be views in the client's statistics menu.")
-@Examples({"set {_t} to time played of player",
+@Examples({
+	"set {_t} to time played of player",
 	"if player's time played is greater than 10 minutes:",
 	"\tgive player a diamond sword",
-	"set player's time played to 0 seconds"})
+	"",
+	"set player's time played to 0 seconds"
+})
+@RequiredPlugins("MC 1.16+ (offline players)")
 @Since("2.5")
```

Comment:
Can you edit the Since annotation to note that offline players were supported since 2.6?